### PR TITLE
fix for #3129

### PR DESCRIPTION
**Pull request by D4imyo**
check the setup.py script then check the prepare() function.

### CODE REVIEW

File: packages/nettacker/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+pkgver=0.0.2.r0.g92239ce
+pkgrel=1
+pkgdesc='automate information gathering, vulnerability scanning and eventually generating a report for networks, including services, bugs, vulnerabilities, misconfigurations, and other information.'
+arch=('any')
+groups=('blackarch' 'blackarch-scanners')
+url='https://github.com/OWASP/Nettacker'
+license=('Apache')
+depends=('flake8' 'python-backoff' 'python-argparse' 'python-netaddr' 'python-ipaddr' 'python-requests' 'python-paramiko' 'python-texttable' 'python-pysocks' 'python-win_inet_pton' 'python-pyopenssl' 'python-flask' 'python-lockfile' 'python-bs4' 'python-pyspf' 'python-sqlalchemy' 'canari' 'python-py3dns' 'python-censys' 'python-six' 'python-shodan' 'python-pycurl' 'python-xmltodict' 'python-rule' 'python-mechanize' 'scapy' 'python-psycopg2')
+makedepends=('python-setuptools' 'git')
+source=("git+https://github.com/OWASP/$_pkgname.git")
```
Comment (noraj) [inline packages/nettacker/PKGBUILD:15]: github URL are case unsensitive so https://github.com/OWASP/$_pkgname.git == https://github.com/OWASP/$pkgname.git, so in the end there is no need for `_pkgname`

File: packages/nettacker/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=nettacker
+_pkgname=Nettacker
+pkgver=0.0.2.r0.g92239ce
+pkgrel=1
+pkgdesc='automate information gathering, vulnerability scanning and eventually generating a report for networks, including services, bugs, vulnerabilities, misconfigurations, and other information.'
```
Comment (noraj) [inline packages/nettacker/PKGBUILD:8]: Please an uppercase on first letter

File: packages/nettacker/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+  python setup.py build
+}
+
+package() {
+  cd $_pkgname
+
+  python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build
+
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
```
Comment (noraj) [inline packages/nettacker/PKGBUILD:41]: No need because Apache is in `/usr/share/licenses/common/`

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/nettacker/PKGBUILD (hunk 1 above)
Comment (D4imyo) [reply packages/nettacker/PKGBUILD:15]: i use it to change my current directory, the directory called Nettacker not nettacker

Comment (noraj) [reply packages/nettacker/PKGBUILD:15]: if you git clone https://github.com/OWASP/nettacker.git the directory will be named nettacker so cd $pkgname will work

Comment (D4imyo) [reply packages/nettacker/PKGBUILD:15]: oh , got it. thanks

File: packages/nettacker/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=nettacker
+pkgver=0.0.2.r0.g92239ce
+pkgrel=1
+pkgdesc='Automate information gathering, vulnerability scanning and eventually generating a report for networks, including services, bugs, vulnerabilities, misconfigurations, and other information.'
```
Comment (ikstream) [inline packages/nettacker/PKGBUILD:7]: Please try to keep the description below 80 character in length. Keep it short and precise

File: packages/nettacker/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+arch=('any')
+groups=('blackarch' 'blackarch-scanners')
+url='https://github.com/OWASP/Nettacker'
+license=('Apache')
+depends=('flake8' 'python-backoff' 'python-argparse' 'python-netaddr' 'python-ipaddr' 'python-requests' 'python-paramiko' 'python-texttable' 'python-pysocks' 'python-win_inet_pton' 'python-pyopenssl' 'python-flask' 'python-lockfile' 'python-bs4' 'python-pyspf' 'python-sqlalchemy' 'canari' 'python-py3dns' 'python-censys' 'python-six' 'python-shodan' 'python-pycurl' 'python-xmltodict' 'python-rule' 'python-mechanize' 'scapy' 'python-psycopg2')
+makedepends=('python-setuptools' 'git')
```
Comment (ikstream) [inline packages/nettacker/PKGBUILD:13]: Break long lines latest at 80 characters here and format the lines nicely. 

For reference you can have a look at `packages/python-taser/PKGBUILD`

**Review by ikstream — CHANGES_REQUESTED** [review]

**Comment by noptrix** [comment]
thank you guys @D4imyo @ikstream @noraj - i will merge this in and do the needed changes. @D4imyo check my changes then afterwards:) thanks again

**Pull request merged**